lib/flv/transmuxer.js

```diff
@@ -442,11 +447,9 @@ Transmuxer = function(options) {
     .pipe(this.metadataStream)
     .pipe(coalesceStream);
   // if CEA-708 parsing is available, hook up a caption stream
-  if (muxjs.mp2t.CaptionStream) {
-    captionStream = new muxjs.mp2t.CaptionStream();
-    h264Stream.pipe(captionStream)
-      .pipe(coalesceStream);
-  }
+  captionStream = new m2ts.CaptionStream();
```

Comment:
This looks like it makes caption parsing a requirement instead of an optional extension like it was previously.